optionals/auto_load_settings_file/XEH_preInit.sqf

```diff
@@ -0,0 +1,12 @@
+#include "script_component.hpp"
+
+ADDON = false;
+
+// frame after preInit, but before postInit
+0 spawn {
```

Comment:
It's a long shot, but maybe some other mod could conceivably clog the scheduler during preinit, preventing this from running before postInit.